app/src/main/java/com/yeonae/chamelezone/view/mypage/myplace/OpeningHoursActivity.kt

```diff
@@ -30,55 +91,55 @@ class OpeningHoursActivity : AppCompatActivity() {
         checkbox_sun.setOnClickListener {
             if (checkbox_sun.isChecked) {
                 addOpeningHourLayout(opening_hour_sun, 1)
-            } else {
+            } else if (!checkbox_sun.isChecked) {
```

Comment:
```suggestion
            } else  {
```
else if로 변경한 이유가 있나요?

Reply:
수정했습니당